src/jvm/test/org/jetbrains/kotlinx/lincheck_test/representation/VarHandleRepresentationTests.kt

```diff
@@ -8,6 +8,8 @@
  * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
+@file:Suppress("Since15")
```

Comment:
Is this suppression necessary?

Reply:
https://github.com/JetBrains/lincheck/pull/416#discussion_r1803876177